examples/aiohttp/starwars/query.json

```diff
@@ -0,0 +1,3 @@
+{
+    "query": "query starwars($episode: String, $humanId: String, $droidId: String) { hero(episode: $episode) { id name friends { id appearsIn } appearsIn } human(id: $humanId) droid(id: $droidId) }", "variables": {"episode": "NEWHOPE", "humanId": "1001", "droidId": "2001"}, "operationName": "starwars"
```

Comment:
is it possible to use multiline to format the GQL query with a more readable indentation?

Reply:
Yes, it is ;)